### ISSUE TITLE
numexpr-py fails to build on 10.13

### ISSUE THREAD

**Issue by kamischi** [fink/fink-distributions#215]
fink build numexpr-pyXX fails to build on 10.13. I tired with py27 and py36. Log:

```
...
/sw/bin/python3.6 setup.py build
running build
running config_cc
unifing config_cc, config, build_clib, build_ext, build commands --compiler options
running config_fc
unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
running build_src
build_src
building py_modules sources
creating build
creating build/src.macosx-10.13-x86_64-3.6
creating build/src.macosx-10.13-x86_64-3.6/numexpr
building extension "numexpr.interpreter" sources
build_src: building npy-pkg config files
running build_py
creating build/lib.macosx-10.13-x86_64-3.6
creating build/lib.macosx-10.13-x86_64-3.6/numexpr
copying numexpr/cpuinfo.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr
copying numexpr/version.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr
copying numexpr/expressions.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr
copying numexpr/__init__.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr
copying numexpr/utils.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr
copying numexpr/necompiler.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr
copying build/src.macosx-10.13-x86_64-3.6/numexpr/__config__.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr
creating build/lib.macosx-10.13-x86_64-3.6/numexpr/tests
copying ./numexpr/tests/__init__.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr/tests
copying ./numexpr/tests/test_numexpr.py -> build/lib.macosx-10.13-x86_64-3.6/numexpr/tests
Skipping optional fixer: buffer
Skipping optional fixer: idioms
Skipping optional fixer: set_literal
Skipping optional fixer: ws_comma
running build_ext
customize UnixCCompiler
customize UnixCCompiler using build_ext
customize UnixCCompiler
customize UnixCCompiler using build_ext
building 'numexpr.interpreter' extension
compiling C++ sources
C compiler: g++ -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -fwrapv -O3 -Wall -I/sw/include

creating build/temp.macosx-10.13-x86_64-3.6/numexpr
compile options: '-I/sw/lib/python3.6/site-packages/numpy/core/include -I/sw/include/python3.6m -c'
extra options: '-funroll-all-loops'
g++: numexpr/interpreter.cpp

Warning: Assuming default configuration (numexpr/tests/{setup_tests,setup}.py was not found)error: Command "g++ -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -fwrapv -O3 -Wall -I/sw/include -I/sw/lib/python3.6/site-packages/numpy/core/include -I/sw/include/python3.6m -c numexpr/interpreter.cpp -o build/temp.macosx-10.13-x86_64-3.6/numexpr/interpreter.o -MMD -MF build/temp.macosx-10.13-x86_64-3.6/numexpr/interpreter.o.d -funroll-all-loops" failed with exit status 1
### execution of /sw/bin/python3.6 failed, exit code 1
### execution of /tmp/fink.KfQGC failed, exit code 1
Removing runtime build-lock...
Removing build-lock package...
/sw/bin/dpkg-lockwait -r fink-buildlock-numexpr-py36-2.6.4-1
(Reading database ... 442337 files and directories currently installed.)
Removing fink-buildlock-numexpr-py36-2.6.4-1 ...
Failed: phase compiling: numexpr-py36-2.6.4-1 failed
...
```
I tried to find some things out, but could not find the issue → sorry, no patch.

**Comment by nieder** [comment]
2.6.6 is the latest upstream and that also fails, but it failed on numexpr/module.c for me.  Adding `--verbose` to the `setup.py --verbose build` command doesn't give any further clues, though I did not look for a log file (only terminal output).

**Comment by kamischi** [comment]
I had a problem, when i tried to create a new description with scikit-learn and scikit-image. Similar fails, with g++ and exit 1 on the first cpp file. I could not really track it down. I found two puzzling details. This is were I got stuck:
1) Executing "python36 setup.py build" on the command line in the source directory actually works fine. It only fails in the CompileScript.
2) the compiler-wrappers from fink in /sw/var/lib/fink/path-prefix-*/ were not executed. No idea, whether this is relevant.
